reana_cluster/backends/kubernetes/templates/deployments/job-controller-template.yaml

```diff
@@ -13,22 +13,31 @@ spec:
       containers:
       - name: job-controller
         image: {{JOB_CONTROLLER_IMAGE}}
-        {% if DEPLOYMENT != 'prod' %}
-        command: ["flask", "run", "-h", "0.0.0.0"]
-        {% endif %}
-        imagePullPolicy: {{IMAGE_PULL_POLICY}}
-        {%- if DEPLOYMENT != 'prod' %}
+        {% if DEBUG %}
+        command: ["/bin/sh","-c"]
+        args: ["python reana_job_controller/app.py"]
```

Comment:
Is `flask run` not working?

Reply:
It's working fine, this was a mistake while rebasing before the change to `flask run`